pxt.json

```diff
@@ -24,7 +24,7 @@
     ],
     "public": true,
     "targetVersions": {
-        "target": "1.3.49",
+        "target": "1.8.22",
```

Comment:
Can we remove the pxt-heap dependency here if we swapped to using a normal array?

(And yeah I definitely believe the array ended up being faster than the heap -- I primarily used my heap because I already had it implemented for fun / as an example for another person and didn't have the time to fully compare the two approaches, but when the total number of values is small and we don't have fully optimized array manipulation the heap implementation loses it's efficacy vs. just iterating)

Reply:
Yes, I agree.
But I can't create suggestion for line 7:
`"pxt-heap": "github:jwunderl/pxt-heap#v0.0.11"`
Shall I remove it in a new PR?

Reply:
Done in 3 commits.